ray-operator/controllers/ray/common/pod.go

```diff
@@ -38,11 +38,14 @@ const (
 	NeuronCoreRayResourceName          = "neuron_cores"
 	TPUContainerResourceName           = "google.com/tpu"
 	TPURayResourceName                 = "TPU"
+	GPUShareContainerResourceName      = "aliyun.com/gpu-mem"
+	GPUShareResourceName               = "gpu_share"
```

Comment:
Is this the actual resource name recognized by Ray? `aliyun.com/gpu-mem` -> `gpu_share` seems like an odd mapping

Reply:
I think this is likely a custom mapping.

Reply:
Maybe naming it `aliyun-gpu` would be better?

Reply:
Apologies, I realized that Ray Resource Name only supports GPU, neuron_cores, TPU, NPU, and HPU. I will mapping aliyun's gpu shae to GPU.

Reply:
yes, I think we should only add custom accelerators here that are in the supported list you referenced

Reply:
I mapped `aliyun.com/gpu-mem` to `num-gpus`.